sqlalchemy_redshift/dialect.py

```diff
@@ -317,6 +319,9 @@ class RedshiftDialect(PGDialect_psycopg2):
     """
 
     name = 'redshift'
+    supports_sequences = False
```

Comment:
Sorry you're going to have to walk me through this. Include these flags where? It's not obvious to me glancing at the test fixtures.